crates/cli/tests/integration_test.rs

```diff
@@ -402,7 +402,7 @@ fn run_fn(r: &mut Runner, func: &str, stdin: Vec<u8>) -> (Vec<u8>, String, u64)
 fn assert_fuel_consumed_within_threshold(target_fuel: u64, fuel_consumed: u64) {
     let target_fuel = target_fuel as f64;
     let fuel_consumed = fuel_consumed as f64;
-    let threshold = 2.0;
+    let threshold = 3.0;
```

Comment:
It's not totally clear why this change is needed, but perhaps I'm missing something?

Reply:
If I don't make this change, the `test_empty` test will always fail on one of Linux or MacOS because the difference between the fuel counts on the different operating systems is 2.3% apart with this change.